lib/chore/new_relic/version.rb

```diff
@@ -1,5 +1,5 @@
 module Chore
   module NewRelic
-    VERSION = "1.0.0"
+    VERSION = "1.0.1"
```

Comment:
FYI I haven't actually published as 1.0.1 yet, to avoid caching issues on Jenkins. Currently this code (minus comment block) is on Gemfury as version 0.dev.0, if you're wanting to test it.